### DISCUSSION TITLE
OSX 10.13.6 version?

### DISCUSSION THREAD

**Discussion by gaorient** [GrandOrgue/grandorgue#2002] [Help]
How far back in the releases do I need to go to find a version that will run on the old High Sierra 10.13.6?

Thanks!

**Comment by oleg68** [comment]
Only SourceForge versions (0.3.1.x) are compatible with that old MacOs version. The first GO version from GitHub (3.2.0) requires at least 10.15.

**Comment by MStraeten** [comment]
you might try to build it yourself using macports or homebrew - but thats experimental because even macports has a release for high sierra the ports itself might have issues with those old macos versions.

**Reply by oleg68** [reply]
Homebrew doesn't work on old MacOs versions.
